### ISSUE TITLE
Composer/Installers requirement out of date

### ISSUE THREAD

**Issue by crazyjaco** [alleyinteractive/wordpress-fieldmanager#825]
I am hitting an issue pulling in fieldmanager via composer because of an incompatibility with the composer/installers requirement.

The fm repo has a semver of ~1.0.
The latest version of composer/installers is 2.0.1 (which we have in our composer file)
https://github.com/composer/installers/releases

Is it possible to raise the requirement to ^2.0 or *?
I realize * is not performant but I also don't want to negatively impact folks that are still on ^1.0 like our repo was until a week ago.

**Comment by crazyjaco** [comment]
This was the error I am getting when trying to update the package.
`    - alleyinteractive/wordpress-fieldmanager 1.0.0 requires composer/installers ~1.0 -> found composer/installers[v1.0.0, ..., 1.x-dev] but it conflicts with your root composer.json require (2.0.1).`

**Comment by crazyjaco** [comment]
composer/installers appears to be compatible with Composer 2.0 starting at version 1.9. If we changed ~1.0 to ^1.0, I think that would suffice. Will update the PR

**Comment by theMikeD** [comment]
I am running into this now too. Evidently this has not yet been merged @dlh01? Or am I doing something wrong?

```
composer require alleyinteractive/wordpress-fieldmanager:^1.5
./composer.json has been updated
Running composer update alleyinteractive/wordpress-fieldmanager
Loading composer repositories with package information                                                                                                                                                                                              Updating dependencies                                 
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - composer/installers v1.0.22 requires composer-plugin-api 1.0.0 -> found composer-plugin-api[2.6.0] but it does not match the constraint.
    - composer/installers[v1.0.23, ..., v1.8.0] require composer-plugin-api ^1.0 -> found composer-plugin-api[2.6.0] but it does not match the constraint.
    - Root composer.json requires alleyinteractive/wordpress-fieldmanager ^1.5 -> satisfiable by alleyinteractive/wordpress-fieldmanager[1.5.0].
    - alleyinteractive/wordpress-fieldmanager 1.5.0 requires composer/installers ~1.0 -> satisfiable by composer/installers[v1.0.0, ..., 1.x-dev].
    - You can only install one version of a package, so only one of these can be installed: composer/installers[v1.0.0, ..., 1.x-dev, v2.0.0-alpha1, ..., 2.x-dev].
    - wpml/sitepress-multilingual-cms 4.5.14 requires composer/installers ^2 -> satisfiable by composer/installers[v2.0.0-alpha1, ..., 2.x-dev].
    - Root composer.json requires wpml/sitepress-multilingual-cms * -> satisfiable by wpml/sitepress-multilingual-cms[4.5.14].


Installation failed, reverting ./composer.json to its original content.

**Comment by theMikeD** [comment]
I don't suppose there is an eta by chance...?

**Comment by dlh01** [comment]
Sorry, just "as soon as we can."

**Comment by dlh01** [comment]
This update is included in [Fieldmanager 1.6](https://github.com/alleyinteractive/wordpress-fieldmanager/releases/tag/1.6.0). Thank you both!

**Comment by theMikeD** [comment]
Thanks David!
